core/src/apps/common/coininfo.py.mako

```diff
@@ -145,6 +151,15 @@ def by_name(name: str) -> CoinInfo:
             ${attr}=${func(coin[attr])},
             % endfor
         )
+% endfor
+    if utils.USE_ZCASH:
```

Comment:
This should probably be `utils.USE_ZCASH or not utils.BITCOIN_ONLY`, or something similar, depending on https://github.com/trezor/trezor-firmware/issues/2375.